React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm

```diff
@@ -33,7 +33,7 @@ @implementation RCTTextInputComponentView {
 #if !TARGET_OS_OSX // [macOS]
   RCTUIView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
 #else // [macOS
-  RCTUITextView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
+  RCTPlatformView<RCTBackedTextInputViewProtocol> *_backedTextInputView;
```

Comment:
Also this one?

Reply:
`contentView` needs to be a `NSView` (aka `RCTPlatformView`) and not a RCTUIView. Most of our native controls do not inherit from `RCTUIView`